cmd/cdi-cloner/cloner_startup.sh

```diff
@@ -51,10 +67,14 @@ if [ "$obj" == "target" ]; then
     while true; do
         echo "cloner: check if the fifo pipe was created by the cloning source pod"
         if [ -e "$pipe_dir" ]; then
-            pushd $image_dir
+            if [ "$volumeMode" != "block" ]; then
```

Comment:
The check if we created the pipe is relevant in all cases no?

Reply:
we do check if the pipe exists. This check is only done before we pushd the directory. As I'm thinking about it now, I do not get exactly why we do this (pushd/popd). The only code we execute between the pushd and popd is '/usr/bin/cdi-cloner -pipedir $pipe_dir -alsologtostderr -v=3'. isn't it leftover from earlier design?

Reply:
I was wrong, the pushd and popd are for switching to the target directory, which is not relevant for block devices.